life_expectancy/tests/conftest.py

```diff
@@ -1,13 +1,17 @@
 """Pytest configuration file"""
 import pandas as pd
 import pytest
+import json
 
 from . import FIXTURES_DIR, OUTPUT_DIR
 
+DATA_LOCATION_CSV = OUTPUT_DIR/"eu_life_expectancy_raw.tsv"
+DATA_LOCATION_JSON = OUTPUT_DIR/"eurostat_life_expect.json"
 
-DATA_LOCATION = OUTPUT_DIR/"eu_life_expectancy_raw.tsv"
 PT_FILE_NAME = OUTPUT_DIR/"pt_life_expectancy_expected.csv"
 
+PT_FILE_NAME_JSON = OUTPUT_DIR/"pt_life_expectancy_expected_json.csv"
```

Comment:
It could be an attrs. Example:
```python
from attr import define, field
@define
class FilePathsTest:
          output_dir: ...
          data_location_csv: str = field(init=False)
          ....
         def __attrs_post_init__(self):
                  self.data_location_csv = self.output_dir/"eu_life_expectancy_raw.tsv"
                  ....
```